src/main/data/facilities.json

```diff
@@ -7089,6 +7089,16 @@
     "code": "WMS",
     "description": "WILD MIKE STUDIO"
   },
+  {
+    "code": "WMX",
+    "contact": {
+      "address": "Av. Pres. Kennedy, 569 - 902 - Boqueirão, Praia Grande - SP, 11701-335, Brazil",
```

Comment:
Is 902 an apartment number?

If so, should the address be formatted as follows:

Av. Pres. Kennedy, 569, apt. 902, Boqueirão, Praia Grande - SP, CEP 11701-335, Brazil

Reply:
The actual entry was: "Av: Presidente Kennedy, 569, sala 902" and "sala" means "room" in Brazil.  So I'm not sure it is an apartment number. Maybe "room" 902?

Reply:
I think we should leave _sala_.